hack/test-end-to-end.sh

```diff
@@ -247,7 +248,7 @@ if [[ "$ROUTER_TESTS_ENABLED" == "true" ]]; then
 
     echo "[INFO] Installing router with master url of ${API_SCHEME}://${CONTAINER_ACCESSIBLE_API_HOST}:${API_PORT} and starting pod..."
     echo "[INFO] To disable router testing set ROUTER_TESTS_ENABLED=false..."
-    "${OS_ROOT}/hack/install-router.sh" "router1" "${API_SCHEME}://${CONTAINER_ACCESSIBLE_API_HOST}:${API_PORT}"
+    OPENSHIFT_CA_DATA=$(cat "${CERT_DIR}/system:openshift-client/root.crt") OPENSHIFT_CERT_DATA=$(cat "${CERT_DIR}/system:openshift-client/cert.crt") OPENSHIFT_KEY_DATA=$(cat "${CERT_DIR}/system:openshift-client/key.key") "${OS_ROOT}/hack/install-router.sh" "router1" "${API_SCHEME}://${CONTAINER_ACCESSIBLE_API_HOST}:${API_PORT}"
```

Comment:
maybe make invocation of `install-router.sh` with certs the same as install-registry.sh? e.g. just pass it a CERT_DIR env var and it looks inside for the right files? That's all going away as soon as possible, so I hate to get people used to calling `install-router.sh` in a really gorpy way.